app/controllers/users_controller.rb

```diff
@@ -1,12 +1,12 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
-    render json: @users
+    render 'index'
```

Comment:
[OPTIONAL]
- [ ] Kindly note that rails automatically render the view that have the same name as the action in the controller. Therefore there is no need to add the `render 'index'` to the `index` action, rails will render `app/views/users/index.html.erb` by default for the `index` action in the users controller.

Reply:
👍